common/Helpers/EnumToIntConverter.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Data;
+
+namespace DevHome.Common.Helpers;
+
+public class EnumToIntConverter : IValueConverter

```

Comment:
Is this class just for converting an into to an enum and an enum to an int?  Why?  Does C# casting functionality lacking in a way?

Reply:
I couldn't get it to work with simple casting, but maybe I didn't do it the right way.

Reply:
o.i.c.  This is a converted for the view.